crates/anvil/core/src/eth/serde_helpers.rs

```diff
@@ -27,7 +27,7 @@ pub mod sequence {
             return Err(serde::de::Error::custom(format!(
                 "expected params sequence with length 1 but got {}",
                 seq.len()
-            )))
```

Comment:
Very dumb question but how to format the code correctly? I've also switched to toolchain in the test ci. It still adds this semicolon everywhere. Not sure what I'm missing

```
cargo fmt --version          
rustfmt 1.7.0-nightly (bf3c6c5b 2024-02-01)
```